src/CozyUtils.js

```diff
@@ -39,6 +43,26 @@ class CozyUtils {
     return get(resp, 'data')
   }
 
+  async findGroups(accountId, remoteIds) {
+    const groupsCollection = this.client.collection(DOCTYPE_CONTACTS_GROUPS)
+    const resp = await groupsCollection.find(
+      {
+        cozyMetadata: {
+          sync: {
+            [accountId]: {
+              id: {
+                $in: remoteIds
+              }
+            }
+          }
+        }
+      },
+      { indexedFields: [`cozyMetadata.sync.${accountId}.id`] }
+    )
```

Comment:
no try/catch/log here?

Reply:
Mmh no, why? There's a larger try catch in the main file, if this produces an error I don't think we can recover from it.